File: src/main/java/me/miki/shindo/management/event/impl/EventKey.java
package me.miki.shindo.management.event.impl;

import me.miki.shindo.management.event.Event;

public class EventKey extends Event {

    private int keyCode;

    public EventKey(int keyCode) {
        this.keyCode = keyCode;
    }

    public int getKeyCode() {
        return keyCode;
    }

    public void setKeyCode(int keyCode) {
        this.keyCode = keyCode;
    }
}

File: src/main/kotlin/me/miki/shindo/management/event/impl/EventKey.kt
package me.miki.shindo.management.event.impl

import me.miki.shindo.management.event.Event

class EventKey(
    private var keyCode: Int,
) : Event() {
    fun getKeyCode(): Int = keyCode

    fun setKeyCode(keyCode: Int) {
        this.keyCode = keyCode
    }
}
